aries-site/src/themes/aries.js

```diff
@@ -1,11 +1,50 @@
 import { hpe } from 'grommet-theme-hpe';
 import { deepMerge } from 'grommet/utils';
+import { Trash } from 'grommet-icons/icons/Trash';
 
 export const aries = deepMerge(hpe, {
   defaultMode: 'dark',
   // To be stripped out once theme changes are made in grommet-theme-hpe
   // keeping file for use as playground for future theme adjusments that need
   // to be quickly tested
+  fileInput: {
```

Comment:
Is this work to add to HPE theme represented in an issue?

Reply:
I will file a PR on the theme with this work.

Reply:
PR is file here on theme: https://github.com/grommet/grommet-theme-hpe/pull/191

Reply:
This theme worked has been merged.